message.go

```diff
@@ -34,7 +35,7 @@ const (
 //
 // Note: Not exported as we might want to support multiple versions
 // transparently in the future.
-const protocolVersion = 2
+const protocolVersion = 6
```

Comment:
Does this break compat with older milters that don't support v6? If so, do we see these in the wild?

Reply:
Yes, it does. Legacy milters may get confused by too high version number. Precise semantics is not documented though - need to study sendmail source to figure out how it behaves. It may be a good idea to make advertised version number changeable.

Reply:
Most milters are based on libmilter. In any supported Linux distro there should be a libmilter version that supports protocol 6 already.

Reply:
OK, then this looks less scary.